command/meta_format.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package command
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/nomad/api"
+)
+
+func (m *Meta) formatNodePoolList(pools []*api.NodePool) string {
```

Comment:
I'm trying to organize these in a sensible place. Not sure if `Meta` is the best option so I open to feedback. Sometimes you need to format a list of something else in a commend (like `nomad job allocs`), so attaching them a specific command doesn't seem right.

Reply:
Ah, I see how it was done for variables. I will copy that pattern then, thanks!